src/bower_components/emby-webcomponents/nowplayingbar/nowplayingbar.js

```diff
@@ -56,7 +56,7 @@ define(['require', 'datetime', 'itemHelper', 'events', 'browser', 'imageLoader',
 
         html += '<button is="paper-icon-button-light" class="muteButton mediaButton"><i class="md-icon">&#xE050;</i></button>';
 
-        html += '<div class="sliderContainer nowPlayingBarVolumeSliderContainer hide" style="width:100px;vertical-align:middle;display:inline-flex;">';
+        html += '<div class="sliderContainer nowPlayingBarVolumeSliderContainer hide" style="width:150px;vertical-align:middle;display:inline-flex;">';
```

Comment:
I really don't like !important. could that be because of this inline style? display inline-flex? perhaps moving this rules to nowplayingbar.css will ease the !important? i could be asking too much already

Reply:
I will check if !important is really important.

Reply:
After some tests, `!important` is needed in this state of code. I will check why